src/fdk_organization_bff/service/org_catalog_service.py

```diff
@@ -86,13 +124,25 @@ async def get_organization_catalog(id: str) -> Optional[OrganizationCatalog]:
             asyncio.ensure_future(fetch_open_licenses(session)),
         )
 
-    results = responses[2].get("results")
-    org_datasets = results.get("bindings") if results else []
-
-    if org_datasets and len(org_datasets) > 0:
+    if responses[2] and len(responses[2]) > 0:
```

Comment:
Fint om du slenger på ein liten kommentar på denne. Kva ligg i `responses[2]`, feks.

Reply:
Tenkte det var veldig klart ut fra opprettelsen av `responses` to linjer ovenfor, men skader vel ikke med en kommentar.

Reply:
77d19d3
Mer lesbart nå?